combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
```

Comment:
Flags are not typically booleans. They are the string that triggered the change in state. Since Ruby is neat, we can use the presence or absence of the string as if it were a boolean.